src/all/misskon/src/eu/kanade/tachiyomi/extension/all/misskon/MissKon.kt

```diff
@@ -0,0 +1,136 @@
+package eu.kanade.tachiyomi.extension.all.misskon
+
+import eu.kanade.tachiyomi.lib.randomua.UserAgentType
+import eu.kanade.tachiyomi.lib.randomua.setRandomUserAgent
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimitHost
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.util.asJsoup
+import keiyoushi.utils.firstInstance
+import keiyoushi.utils.tryParse
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.concurrent.TimeUnit
+
+class MissKon() : SimpleParsedHttpSource() {
+
+    override val baseUrl = "https://misskon.com"
+    override val lang = "all"
+    override val name = "MissKon"
+    override val supportsLatest = true
+
+    override val client = network.cloudflareClient.newBuilder()
+        .rateLimitHost(baseUrl.toHttpUrl(), 10, 1, TimeUnit.SECONDS)
+        .setRandomUserAgent(UserAgentType.MOBILE)
+        .build()
+
+    override fun simpleMangaSelector() = "article.item-list"
+
+    override fun simpleMangaFromElement(element: Element): SManga {
+        val titleEL = element.selectFirst(".post-box-title")!!
+        return SManga.create().apply {
+            title = titleEL.text()
+            thumbnail_url = element.selectFirst(".post-thumbnail img")?.absUrl("data-src")
+            setUrlWithoutDomain(titleEL.selectFirst("a")!!.absUrl("href"))
+        }
+    }
+
+    override fun simpleNextPageSelector(): String? = null
+
+    // region popular
+    override fun popularMangaRequest(page: Int) = GET("$baseUrl/top3/", headers)
+    // endregion
+
+    // region latest
+    override fun latestUpdatesRequest(page: Int) = GET("$baseUrl/page/$page", headers)
+
+    override fun latestUpdatesNextPageSelector() = ".current + a.page"
+    // endregion
+
+    // region Search
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val filter = filters.firstInstance<SourceCategorySelector>()
+        return filter.selectedCategory?.let {
+            GET("$baseUrl$it.url", headers)
```

Comment:
```suggestion
            GET("$baseUrl${it.url}", headers)
```
you are putting literal string "url" in the url?